bliss.shy.js

```diff
@@ -628,11 +628,11 @@ $.Element.prototype = {
 				type = type[0];
 			}
 
-			if (type && options.callback) {
-				return $.original.removeEventListener.call(this, type, options.callback, options.capture);
-			}
-
+			//if listeners exist, always go through listeners to clean up
```

Comment:
Could you help me understand what the memory leak was here? Is this change for the capture flag, in case there are two listeners with the same type and callback but different captures? 
Because if all params are the same, the browser deduplicates it automatically, so not sure why we need to go through listeners, but I'm probably missing something since I haven't looked into it as closely!.